src/ni/vsbuild/Pipeline.groovy

```diff
@@ -39,6 +39,13 @@ class Pipeline implements Serializable {
          stages << new Package(script, buildConfiguration, lvVersion)
       }
 
+      def shouldBuildPackage() {
+         return buildConfiguration.packageInfo &&
+            (script.env.BRANCH_NAME.startsWith("release") ||
+            script.env.BRANCH_NAME.startsWith("hotfix") ||
+            script.env.BRANCH_NAME == "master")
+      }
```

Comment:
Is this the best place for this logic to live? The policy regarding which branches to build packages for seems like it should exist outside of the build pipeline itself, as a configuration instead of implementation detail.

Reply:
> Is this the best place for this logic to live? The policy regarding which branches to build packages for seems like it should exist outside of the build pipeline itself, as a configuration instead of implementation detail.

The plan is to limit the build pipeline to release and hotfix branches. All changes are submitted to master which is then merged to release branch. We build packages and installers from the release branch using the version number of the release branch.

Reply:
Took master out since automatic merging is coming soon.